### ISSUE TITLE
Language Profile Images

### ISSUE THREAD

**Issue by sethmcknight** [SeedCompany/cord-api-v3#1237]
- Display on Language Card and Language Profile
- Default Language Images by Location (I can get these for you based on what's on sc.com)
- Allow users to upload a specific image to replace the default

**Comment by CarsonF** [comment]
The big question here is whether we create a signed url every time the picture is asked for, making it private. Or making it public, which would allow it to be cached and load faster.

**Comment by glassblowerscat** [comment]
> The big question here is whether we create a signed url every time the picture is asked for, making it private. Or making it public, which would allow it to be cached and load faster.

How hard would it be to make the default ones public and the customized ones private? I'm guessing customization would be fairly rare, but I don't really know our user base that well

**Comment by CarsonF** [comment]
Yeah fallback logic is another thing to work through. Just like our other override fields, we need to know if there's actually been a picture set or not, separate from the field that encompasses the fallback.

SIgned urls are going to function very poorly in a list view.
